### DISCUSSION TITLE
Mentor Discussion Channel

### DISCUSSION THREAD

**Discussion by sanjay-kv** [recodehive/Stackoverflow-Analysis#86] [General]
This would be the mentor dissection channel, Please introduce yourself.

**Comment by sanjay-kv** [comment]
@okaditya84 This will be the guidelines.

- As you see many people comment at a particular issue, so make sure you copy paste the below there.

"Issue will be assigned to Only 1 person ...FCFS basis. 
Others get a chance if the issue gets stale & the assignee is inactive for 5+ days
Also if you would like to work on more issues create a new issue and I will create the label and assign you."

- Detail about which issue to be assigned I will make a video and sent to you.
- Most of the repetitive task has been automated with GitHub Action in this repo.
- We are focusing on two files , Streamlit and main ipynb files. 
- Make sure to assign the labels, have a good collab with contributors, and then make discussion board active.

**Reply by sanjay-kv** [reply]
@Princegupta101 i will give you access

**Comment by AKSHITHA-CHILUKA** [comment]
>Hii I am Akshitha one of the contibutor in gssoc'24

**Comment by marv1n-le** [comment]
Make sure to assign the labels, have a good collab with contributors, and then make discussion board active.
